Kube/scripts/install-shield.sh

```diff
@@ -382,27 +386,41 @@ if [ ! -f ~/.kube/config ] || [ $(cat ~/.kube/config | wc -l) -le 1 ]; then
     fi
 
     step
-
+    
```

Comment:
This can be also skipped in offline, since it is done by ova preparation (ansible)

Reply:
but it doesnt hurt...